walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
```

Comment:
Just to double check, this means I should change "import org.junit.Assert;" to "import static org.junit.Assert;" and this will allow me to call assertEquals instead of Assert.assertEquals ?

Reply:
Yes. 

```Static imports are appropriate when the usage is clear without additional context of the enclosing type. Static imports serve to remove redundant context and reduce boilerplate. The method/field should be fairly well-known, and self-describing.```

Here Assert.assertEquals are very well-known and self-describing so most of the time people will use static import.

Reply:
Gotcha.  The makers of this walkthrough wrote  most of these test functions but good to know and I will make the changes!